ubuntu16/tasks/main.yml

```diff
@@ -1,7 +1,7 @@
 ---
 - name: "Check for version 16.04"
   fail: msg="Must be version 16.04 or higher"
-  when: ansible_distribution_version | version_compare('16.04', '<')
+  when: ansible_distribution_version | version_compare('16.04', '>=')
```

Comment:
i need to check this, the intent of the line above is to fail when the version is less than 16.04, not the other way around. (eg: fail when true, not fail when false). o/w this would bomb out on me for the normal release testing (?)

http://docs.ansible.com/ansible/latest/modules/fail_module.html

i could be missing something else though too- sometimes the ansible vars don't act the way you'd expect them, want to make sure it doesn't break the other way around since it's passing right now.

Reply:
No problem.

Tested the build with my fork by the way (which has the changes) and it built successfully.

Reply:
```
root@7436cb529ce1:/home/cif# cif --version
3.0.0rc1
```

That said, I am seeing other issues too:
```
root@7436cb529ce1:/# cif -p
Traceback (most recent call last):
  File "/usr/local/bin/cif", line 11, in <module>
    load_entry_point('cifsdk==3.0.0rc1', 'console_scripts', 'cif')()
  File "/usr/local/lib/python2.7/dist-packages/cifsdk/client/client.py", line 142, in main
    ret = cli.ping()
  File "/usr/local/lib/python2.7/dist-packages/cifsdk/client/http.py", line 227, in ping
    rv = self._get(uri)
  File "/usr/local/lib/python2.7/dist-packages/cifsdk/client/http.py", line 97, in _get
    raise e
cifsdk.exceptions.AuthError: unauthorized
```

But I am guessing that's around the generated token and the placement of it.

Reply:
```
default: TASK [Check for version 16.04] *************************************************
    default: task path: /vagrant/deploymentkit/ubuntu16/tasks/main.yml:2
    default: fatal: [localhost]: FAILED! => {"changed": false, "failed": true, "msg": "Must be version 16.04 or higher"}
```

something else is going on because that change breaks the default install.

Reply:
Let me do a clean pull of your repository just to make sure I don't have any other adds.

Reply:
That's a really good idea (and I wonder if that's what ends up happening basically by reversing the check).

Would you be up for adding that?

Reply:
@wesyoung - cancelling this PR -- I started clean from your latest release, and a clean repo on my side just to make sure that everything is in sync.